### PR TITLE
Zohosign ms powerautomate change

### PR DESCRIPTION
**Pull request by ZohoSign-integ**
---
Please check the following conditions for your PR.

- [x] `apiDefinition.swagger.json` is validated using `paconn validate` command.
- [x] `apiProperties.json` has a valid brand color. Invalid brand colors are `#007ee5` and `#ffffff`.

### CODE REVIEW

**Comment by ptalrejagithub** [comment]
@ZohoSign-integ, can you please just submit the apiDefinationswagger.json, apiProperties.json and readme,md. The readme.md file is required for open sourcing your connector which can be helpful to the open-source users who may want to contribute in your connector. A reference of a readme.md file is [here](https://github.com/st0014/PowerPlatformConnectors/blob/dev/custom-connectors/AzureKeyVault/Readme.md)

**Comment by sriyen-msft** [comment]
@ZohoSign-integ Please remove your connector files from `custom-connectors` folder and update the PR

**Review by ZohoSign-integ — COMMENTED** [review]
removed the zohosign files under custom connector and updated the readme file also.

**Comment by ZohoSign-integ** [comment]
Hi Team,



I have corrected and updated the PR. Please review. 



https://github.com/microsoft/PowerPlatformConnectors/pull/843#pullrequestreview-669761314



Regards,

Gopi









---- On Thu, 27 May 2021 03:43:49 +0530 Srikanth Yenagandhula ***@***.***> wrote ----




https://github.com/ZohoSign-integ Please remove your connector files from custom-connectors folder and update the PR

File: certified-connectors/ZohoSign/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,1943 @@
...
+        "description": "Invite users with email id",
+        "consumes": [
+          "application/x-www-form-urlencoded"
+        ],
+        "parameters": [
+          {
+            "in": "formData",
+            "name": "email_id",
+            "type": "string",
+            "description": "[\"Enter the email ids of all the invitees whom you want to to be added\"]",
```
Comment (sriyen-msft) [inline certified-connectors/ZohoSign/apiDefinition.swagger.json:210]: @ZohoSign-integ Can you please let us knw why this description is double quotes

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/ZohoSign/apiDefinition.swagger.json (hunk 1 above)
Comment (ZohoSign-integ) [reply certified-connectors/ZohoSign/apiDefinition.swagger.json:210]: It is to highlight the sentence inside the square brackets, for better understanding.

File: certified-connectors/ZohoSign/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,1943 @@
...
+                            false
+                          ]
+                        },
+                        "branding": {
+                          "type": "boolean",
+                          "description": "branding"
+                        },
+                        "custom_domains": {
+                          "type": "boolean",
+                          "description": "custom_domains"
```
[the remark this answers is not in the document]
Comment (ZohoSign-integ) [reply certified-connectors/ZohoSign/apiDefinition.swagger.json]: all the descriptions are corrected. Please review them @sriyen-msft

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**